### PR TITLE
nerfs parrying bullets with a wooden stick following complaints

### PR DESCRIPTION
**Pull request by silicons**
drops bokken efficiency to 30% from 100%.

### CODE REVIEW

File: code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/obj/item/banhammer
+w/obj/item/banhammer
```
Comment (lolman360) [inline code/game/objects/items/weaponry.dm]: w/obj/item/banhammer

**Comment by necromanceranne** [comment]
The ultimate dark souls experience. Losing to a wooden spoon wielding naked lunatic.

**Comment by shellspeed1** [comment]
It just doesn't make sense for a wooden stick to be able to parry exosuit laser cannons, beam rifles, tasers, shotguns, and god knows what else.

**Comment by necromanceranne** [comment]
It doesn't, but it is fucking hilarious and deeply humbling.

**Review by Linzolle — APPROVED** [review]

**Comment by Hatterhat** [comment]
damn

**Pull request merged**